controllers/crypto/usersController.js

```diff
@@ -0,0 +1,186 @@
+const logger = require('../../utils/logger')
+const userQuery = require('../../models/crypto/users')
+
+/**
+ * Fetches the data of user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getUser = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser(req.params.id)
+
+    if (result) {
+      return res.status(200).json({
+        message: 'User returned successfully!',
+        user: result.user
+      })
+    }
+
+    return res.boom.notFound('User doesn\'t exist')
+  } catch (error) {
+    logger.error(`Error while fetching user: ${error}`)
+    return res.boom.badImplementation('Something went wrong please contact admin')
+  }
+}
+
+/**
+ * Add new user
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - user object
+ * @param res {Object} - Express response object
+ */
+const addNewUser = async (req, res) => {
```

Comment:
These functions to add new user, get user are already present 🤔. Why are they required here?